values.yaml

```diff
@@ -75,7 +75,12 @@ global:
   defaultDenyNetworkPolicy: true
 
   # Enable security context constraints required for OpenShift
-  sccEnabled: false
+  sccEnabled: true
+
+  # Deploy auth sidecar to use openshift native features
+  sidecar:
+    enabled: true
```

Comment:
default is false I suppose

Reply:
seems i have added my local values.yaml changes

Reply:
updated values.yaml and pushed the changes with default values